src/DIRAC/Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -197,7 +198,7 @@ def __writeSub(self, executable, nJobs, location, processors):
 output = $(Cluster).$(Process).out
 error = $(Cluster).$(Process).err
 log = $(Cluster).$(Process).log
-environment = "HTCONDOR_JOBID=$(Cluster).$(Process)"
+environment = "HTCONDOR_JOBID=$(Cluster).$(Process) DIRAC_PILOT_STAMP=$(stamp)"
```

Comment:
is `stamp` defined?

Reply:
`stamp` is defined in the `Queue stamp...` below